package.json

```diff
@@ -113,19 +113,23 @@
     "@ckeditor/ckeditor5-slash-command": "40.0.0",
     "@ckeditor/ckeditor5-template": "40.0.0",
     "@ckeditor/ckeditor5-track-changes": "40.0.0",
+    "@ocular-d/vale-bin": "2.29.1",
```

Comment:
Why is this file locked to the specific version while other dev deps use caret ranges here?